portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,59 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
+
 /**
- * Adds a random greeting to the page.
+ * Adds a random hobby to the page by displaying the hobby's description text
+ * and updating the wrapper's background color so that it appears 
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomHobby() {
+  const hobbies =
+      ['playing the Clarinet', 'reading mystery novels', '(trying to) speak Spanish', 'making pasta from different cuisine'];
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const hobby = hobbies[Math.floor(Math.random() * hobbies.length)];
```

Comment:
What do you think about making so it can't pick the same hobby twice in a row? The user experience is a little awkward if you click and it doesn't change. Makes it feel like something is broken.